tox.ini

```diff
@@ -14,3 +14,20 @@ deps = pytest
        pytest-cov
        pytest-cache
 commands = {envpython} setup.py test
+
+
+# Add the following to your ~/.pypirc:
+#   [pypitest]
+#   repository = https://testpypi.python.org/pypi
+#   username = <user>
+#   password = <password>
+# For releasing, simply remove "-r pypitest".
```

Comment:
Well, then at least a reference to https://wiki.python.org/moin/TestPyPI since it will failed without being configured properly.